packages/guides/src/Nodes/DocumentTree/SectionEntryNode.php

```diff
@@ -2,13 +2,13 @@
 
 declare(strict_types=1);
 
-namespace phpDocumentor\Guides\Meta;
+namespace phpDocumentor\Guides\Nodes\DocumentTree;
 
 use phpDocumentor\Guides\Nodes\TitleNode;
 
-class SectionEntry implements ChildEntry
+class SectionEntryNode
```

Comment:
This class has the suffix `Node` but it's not a node?

Reply:
I think I changed that in the next Pull Request where I noticed it when I could not transform it. But changed it now